lib/library.rb

```diff
@@ -0,0 +1,63 @@
+require 'date'
+require 'yaml'
+
+class Library
+
+  RETURN_DURATION = 1   # Value is in month
+
+  attr_accessor :books, :borrowed_book
+
+  def initialize
+
+    @books = YAML.load_file('./lib/data.yml')
+
+  end
+
+  def books_instock
+    @books.select { |book| book[:available] == true }
+  end
+
+  def book_available?(title, author)
+    search_book(title, author)
+  end
+
+  def book_checkout(title, author)
+    checkout(title,author)
+  end
+
+  def search_book(title, author)
+    search(title, author)
+  end
+
+  private
+
+  def search(title, author)
```

Comment:
Why did you choose to make a private method `search` that get's called in `seach_book`?